releases/release-7.0.0.md

```diff
@@ -0,0 +1,468 @@
+>
+> This section provides compatibility changes you need to know when you upgrade from v6.6.0 to the current version (v7.0.0). If you are upgrading from v6.5.0 or earlier versions to the current version, you might also need to check the compatibility changes introduced in intermediate versions.
+
+### MySQL compatibility
+
+* TiDB removes the constraint that the auto-increment column must be an index [#40580](https://github.com/pingcap/tidb/issues/40580) @[tiancaiamao](https://github.com/tiancaiamao) **tw:ran-huang**
+
+    Starting from v7.0.0, TiDB removes the constraint that the auto-increment column must be an index or index prefix. Now you can define the primary key of a table more flexibly and use the auto-increment column to implement sorting and pagination more conveniently. This also avoids the write hotspot problem caused by the auto-increment column and improves query performance by using the table with clustered indexes. Previously, TiDB's behavior was consistent with MySQL, requiring the auto-increment column to be an index or index prefix. With the new release, you can create a table using the following syntax and successfully remove the auto-increment column constraint:
+
+    ```sql
+    CREATE TABLE test1 (
+        `id` int(11) NOT NULL AUTO_INCREMENT,
+        `k` int(11) NOT NULL DEFAULT '0',
+        `c` char(120) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+        PRIMARY KEY(`k`, `id`)
+    );
+    ```
+
+    This feature does not affect TiCDC data replication.
+
+    For more information, see [documentation](/mysql-compatibility.md#auto-increment-id).
+
+* TiDB supports Key partitions, as shown in the following example: **tw:qiancai**
+
+    ```sql
+    CREATE TABLE employees (
+    id INT NOT NULL,
+    fname VARCHAR(30),
+    lname VARCHAR(30),
+    hired DATE NOT NULL DEFAULT '1970-01-01',
+    separated DATE DEFAULT '9999-12-31',
+    job_code INT,
+    store_id INT) PARTITION BY KEY(store_id) PARTITIONS 4;
+    ```
+
+    Starting from v7.0.0, TiDB supports Key partitions and can parse the MySQL `PARTITION BY LINEAR KEY` syntax. However, TiDB ignores the `LINEAR` keyword and uses a non-linear hash algorithm instead. Currently, the `KEY` partition type does not support partition statements with an empty partition column list.
+
+    For more information, see [documentation](/partitioned-table.md#key-partitioning).
+
+### TiCDC compatibility
+
+* TiCDC fixes the issue of incorrect encoding of Float data in Avro [#8490](https://github.com/pingcap/tiflow/issues/8490) @[3AceShowHand](https://github.com/3AceShowHand) **tw:ran-huang**
+
+    When upgrading the TiCDC cluster to v7.0.0, if a table replicated using Avro contains the Float data type, you need to manually adjust the compatibility policy of Confluent Schema Registry to `None` before upgrading so that the changefeed can successfully update the schema. Otherwise, after upgrading, the changefeed will be unable to update the schema and enter an error state.
```

Comment:
```suggestion
* TiCDC fixes the issue of incorrect encoding of `FLOAT` data in Avro [#8490](https://github.com/pingcap/tiflow/issues/8490) @[3AceShowHand](https://github.com/3AceShowHand) **tw:ran-huang**

    When upgrading the TiCDC cluster to v7.0.0, if a table replicated using Avro contains the `FLOAT` data type, you need to manually adjust the compatibility policy of Confluent Schema Registry to `None` before upgrading so that the changefeed can successfully update the schema. Otherwise, after upgrading, the changefeed will be unable to update the schema and enter an error state.
```